cloudant-sync-datastore-android/src/test/java/com/cloudant/android/TestReplicationService.java

```diff
@@ -25,6 +25,8 @@
     private static final String DATASTORE_MANGER_DIR = "data";
     private static final String TAG = "TestReplicationService";
 
+    private int mUnboundIntervalSeconds = 120;
```

Comment:
Although this value already existed before this PR, is adding a comment worthwhile? is it documented elsewhere?